src/components/storage/ModifyStorage.jsx

```diff
@@ -29,19 +29,24 @@ import { WrenchIcon, ExternalLinkAltIcon } from "@patternfly/react-icons";
 const _ = cockpit.gettext;
 const N_ = cockpit.noop;
 
-const startBlivetGUI = (onStart, onStarted, errorHandler) => {
-    console.log("Spawning blivet-gui.");
-    // We don't have an event informing that blivet-gui started so just wait a bit.
-    const timeoutId = window.setTimeout(onStarted, 3000);
-    cockpit.spawn(["blivet-gui", "--keep-above", "--auto-dev-updates"], { err: "message" })
-            .then(() => {
-                console.log("blivet-gui exited.");
-                // If the blivet-gui exits earlier cancel the delay
-                window.clearTimeout(timeoutId);
-                return onStarted();
-            })
-            .catch((error) => { window.clearTimeout(timeoutId); errorHandler(error) });
+let cockpit_window = null;
+
+const startCockpitStorage = (onStart, onStarted, errorHandler) => {
+    window.localStorage.setItem("cockpit_anaconda",
+                                JSON.stringify({
+                                    mount_point_prefix: "/mnt/sysimage",
+                                    ignore_devices: ["/dev/sr0", "/dev/loop0"]
```

Comment:
This has changed to `available_devices` and Anaconda could probably easily fill this now with real information from the backend.